packages/vscode-extension/src/utilities/platform.ts

```diff
@@ -0,0 +1,21 @@
+import * as os from "os";
+
+export class Platform {
```

Comment:
I kinda don't like the fact that we renamed Platform to DevicePlatform since it's used way more frequently. I'd see this named `NativePlatform` or `HostPlatform` and potentially shared with webview in global variable instead of having two classes for it.